docker-compose.yaml

```diff
@@ -141,6 +142,11 @@ services:
     environment:
       - "MINIO_ROOT_USER=${DIFFGRAM_MINIO_ACCESS_KEY_ID}"
       - "MINIO_ROOT_PASSWORD=${DIFFGRAM_MINIO_ACCESS_KEY_SECRET}"
+    healthcheck:
```

Comment:
`createbuckets` task still fails for me sometimes with `mc: <ERROR> Unable to initialize new alias from the provided credentials. Server not initialized, please try again.` error. I'm looking into making a better healthcheck to try to address it.